diff --git a/node_modules/react-native-reanimated/android/CMakeLists.txt b/node_modules/react-native-reanimated/android/CMakeLists.txt
index 6300ffb..7137ead 100644
--- a/node_modules/react-native-reanimated/android/CMakeLists.txt
+++ b/node_modules/react-native-reanimated/android/CMakeLists.txt
@@ -37,11 +37,11 @@ set(NATIVEVIEW_DIR "${CMAKE_SOURCE_DIR}/../Common/NativeView")
 set(PACKAGE_ROOT_DIR "${CMAKE_SOURCE_DIR}/..")
 set(CODEGEN_OUTPUT_DIR "${BUILD_DIR}/generated/source/codegen/jni")
 
-file(GLOB_RECURSE REANIMATED_COMMON_CPP_SOURCES CONFIGURE_DEPENDS
+file(GLOB_RECURSE REANIMATED_COMMON_CPP_SOURCES
      "${COMMON_CPP_DIR}/reanimated/*.cpp")
-file(GLOB_RECURSE REANIMATED_ANDROID_CPP_SOURCES CONFIGURE_DEPENDS
+file(GLOB_RECURSE REANIMATED_ANDROID_CPP_SOURCES
      "${ANDROID_CPP_DIR}/reanimated/*.cpp")
-file(GLOB_RECURSE REANIMATED_NATIVEVIEW_CPP_SOURCES CONFIGURE_DEPENDS
+file(GLOB_RECURSE REANIMATED_NATIVEVIEW_CPP_SOURCES
      "${NATIVEVIEW_DIR}/react/renderer/components/rnreanimated/*.cpp")
 
 find_package(fbjni REQUIRED CONFIG)
